code/src/cljs/sixsq/nuvla/ui/edges/subs.cljs

```diff
@@ -127,10 +127,15 @@
   ::nuvlabox-releases-options
   :<- [::nuvlabox-releases]
   (fn [nuvlabox-releases]
-    (map
-      (fn [{:keys [id release pre-release]}]
-        {:key release, :text (str release (when pre-release " - pre-release")), :value id, :pre-release pre-release})
-      nuvlabox-releases)))
+    (let [published-flags-exists? (some :published nuvlabox-releases)
+          nuvlabox-releases         (if published-flags-exists?
```

Comment:
I think we need to show everything visible to the user. Why filter by published?